### PR TITLE
Issue #732 deployment modal dialog support cpu ram disk

### PR DESCRIPTION
**Pull request by 0xbase12**
Connected to #732

### CODE REVIEW

**Comment by loomis** [comment]
When deploying your changes on a v3.29 SlipStream, I have the following problems with the [component deployment dialog](https://cloud.githubusercontent.com/assets/490506/26722516/9a120092-4790-11e7-856d-73d25f30f21e.png):

 1. Titles for the CPU/RAM/Disk fields are not shown
 2. The "More" link doesn't work.

For an [application deployment dialog](https://cloud.githubusercontent.com/assets/490506/26722519/9be54780-4790-11e7-8014-24f6132f7849.png):

 1. The "More link doesn't work.

I see the same issues with both Safari and Chrome. 

Please also merge the changes in master into the PR to avoid missing them.

**Comment by loomis** [comment]
@0xbase12 BTW, I won't be available to retest this until next Wednesday.   If you want it merged before then, you can ask someone else to review it or merge it yourself.

**Comment by 0xbase12** [comment]
@loomis Thanks for feedback. Nothing urgent,I checked now in a deployed SS.
The issue come from that the old receipts does not have cpu.nb ram.GB and disk.GB.
A simple edit save on a module resolve the problem.
I should take into account if old receipts are being used.

**Comment by loomis** [comment]
@0xbase12 Should I test this PR again?

File: clj/resources/static_content/js/module.js [hunk 1]
```diff
@@ -529,6 +529,7 @@ jQuery( function() { ( function( $$, $, undefined ) {
         });
 
         $btnRefreshPRS.click(function() {
+            cachedPRSResponse = null;
```
Comment (loomis) [inline clj/resources/static_content/js/module.js]: I believe that there is also caching/memoization of the PRS responses on the server side.  You may also want to check if it in fact does this and if so remove the memoization.

File: clj/src/slipstream/ui/models/module/deployment.clj [hunk 2]
```diff
@@ -24,18 +25,24 @@
...
-     :default-multiplicity  (-> attrs :multiplicity uc/parse-pos-int (or 1))
-     :default-cloud         (->> (-> attrs :cloudService (or "default")) (u/enum cloud-names :cloud-names))
-     :output-parameters     (->> (html/select node-metadata [:image :parameters :entry [:parameter (html/attr-has :category "Output")]])
-                                 (map (comp :name :attrs))
-                                 sort)
-     :mappings              (cond->> (html/select node-metadata [:parameterMappings :entry :parameter])
-                              template-node?  append-template-mapping
-                              :always         (mapv parse-parameter-mapping)
-                              :always         (sort-by :name))}))
+    (println )
```
Comment (schaubl) [inline clj/src/slipstream/ui/models/module/deployment.clj]: @0xbase12 what is this ?

Comment (0xbase12) [inline clj/src/slipstream/ui/models/module/deployment.clj]: A forgoten debug line

**Pull request merged**